configure.php

```diff
@@ -653,22 +653,20 @@ function install_plugin( array $plugin_data, bool $prompt, &$installed_plugins )
 		run( 'ln -s mu-plugins/drop-ins/object-cache.php object-cache.php' );
 	}
 
-	write( 'Scaffolding out vip-config...' );
-
-	run( 'mkdir -p vip-config && touch vip-config/.gitkeep' );
-
 	write( 'Scaffolding out VIP directories...' );
 
 	run( 'mkdir -p images && touch images/.gitkeep' );
 	run( 'mkdir -p languages && touch languages/.gitkeep' );
 	run( 'mkdir -p private && touch private/.gitkeep' );
+	run( 'touch vip-config/.gitkeep' );
```

Comment:
is the vip-config dir being created at some other point? if it doesn't exist, `touch` will trigger an error, whereas the command above `mkdir -p vip-config && touch vip-config/.gitkeep` will create the dir if it doesn't exist and it will not throw an error if the dir already exists. It is probably safer to use this over just the `touch` command.

```bash
jake@~$ touch testing-touch/myfile.txt
touch: testing-touch/myfile.txt: No such file or directory

Reply:
```suggestion
	run( 'mkdir -p vip-config && touch vip-config/.gitkeep' );
```